### ISSUE TITLE
Colorblind people can not see the difference between trees

### ISSUE THREAD

**Issue by SuperMeepBoy** [adil192/ricochlime#25]
Last update changed slimes for trees. I can not differenciate them now because I am colorblind.

It is kind of related to an other accessibility issue except I don't have my screen turned in black and white so I can never see the difserence, hence the ticket.

**Comment by adil192** [comment]
This is a duplicate of #8, the black and white thing was just a way to demonstrate the issue for colorblind people. 

I'm having some trouble deciding how to show the difference between the monsters. Do you have any ideas?

**Comment by SuperMeepBoy** [comment]
> This is a duplicate of #8, the black and white thing was just a way to demonstrate the issue for colorblind people.
> 
> I'm having some trouble deciding how to show the difference between the monsters. Do you have any ideas?

I think the best way is for them to have different styles (shapes and colors). Or one could be unicolor while the other could look like a birch (white and brown/black).
